src/server/index.jsx

```diff
@@ -293,16 +294,26 @@ server
           routes,
           urlPath,
         );
-        const data = await route.getInitialData({
+
+        const dataPromise = route.getInitialData({
           path: url,
           service,
           variant,
         });
+
+        const remoteConfigPromise = getRemoteConfig(service);
```

Comment:
Move this above the getInitialData call and don't call in parallel - we want to be able to pass these config values into getInitialData.